app/serializers/item_serializer.rb

```diff
@@ -0,0 +1,15 @@
+class ItemSerializer < ActiveModel::Serializer
+  attributes :id,
+             :name,
+             :unit_price,
+             :description,
+             :merchant,
+             :invoice_count,
+             :created_at,
+             :updated_at,
+             :dollar_price
```

Comment:
@stovermc Yes, but I don't think the spec wants :created_at or :updated_at. If you are opening up all attributes there's no need for a serializer.